activerecord/lib/active_record/database_configurations/hash_config.rb

```diff
@@ -54,6 +54,10 @@ def host
         configuration_hash[:host]
       end
 
+      def socket
```

Comment:
Let's `# :nodoc:` this one.

```suggestion
      def socket # :nodoc:
```